### PR TITLE
feat: trap focus to the content & focus first element

### PR DESCRIPTION
**Pull request by anuraghazra**

### CODE REVIEW

**Review by navin-moorthy — APPROVED** [review]

File: script.js [hunk 1]
```diff
@@ -4,18 +4,49 @@ window.addEventListener("DOMContentLoaded", function () {
...
-  new SwipeableBottomSheet({ triggerId: "trigger-2", bottomSheetId: "swipeable-bottom-sheet-2" });
-  new SwipeableBottomSheet({ triggerId: "trigger-3", bottomSheetId: "swipeable-bottom-sheet-1", overlay: false });
-  new SwipeableBottomSheet({ triggerId: "trigger-10", bottomSheetId: "swipeable-bottom-sheet-2", overlay: false });
-  new SwipeableBottomSheet({ triggerId: "trigger-4", bottomSheetId: "swipeable-bottom-sheet-2", peek: "50vh" });
+  new SwipeableBottomSheet({
+    triggerId: "trigger-1",
+    bottomSheetId: "swipeable-bottom-sheet-1",
+    onOpen: handleOnOpen,
+  });
+  new SwipeableBottomSheet({
```
Comment (anuraghazra) [inline script.js:12]: Sorry about this formatting, vscode auto formatted it.

Comment (navin-moorthy) [reply script.js:12]: No worries, I have to add this tooling & build setup with Parcel anyways
https://github.com/timelessco/scripbox-mf

I will do it this weekend.

**Review by navin-moorthy — APPROVED** [review]
LGTM !!

**Comment by navin-moorthy** [comment]
Great Work !!

**Pull request merged**